### DISCUSSION TITLE
Lesson 7 FundMeTest

### DISCUSSION THREAD

**Discussion by Robschuh** [Cyfrin/foundry-full-course-cu#1858] [General]
Hi there, 
I am trying to understand the result of the test "forge test --match-test testPriceFeedVersionAccurate -vvv"
In my case, there are two outputs (as you can see in the attached image) : **0x5fb**... and **0x9d9** .... what is the reason for this? 
NOTE: I can see in  sepolia that both are contracts, but why two contracts and now only one?
https://sepolia.etherscan.io/address/0x9d9305445F404E925563d5D5EcC65C815Ec1655b
https://sepolia.etherscan.io/address/0x5fb1616F78dA7aFC9FF79e0371741a747D2a7F22

![doubt-tests01](https://github.com/Cyfrin/foundry-full-course-f23/assets/142880927/34982047-8f4c-426d-99da-115b0facfbd1)

**Comment by urosognjenovic** [comment]
At the address https://sepolia.etherscan.io/address/0x9d9305445F404E925563d5D5EcC65C815Ec1655b, the function version:
```
function version()
    external
    view
    override
    returns (uint256)
{
    return currentPhase.aggregator.version();
}
```
calls the ```version()``` function at the address https://sepolia.etherscan.io/address/0x5fb1616F78dA7aFC9FF79e0371741a747D2a7F22, which has this line of code:
```
uint256 constant public override version = 4;
```
That's why there are two calls, one from ```FundMe``` contract to the ```version``` function at the address https://sepolia.etherscan.io/address/0x9d9305445F404E925563d5D5EcC65C815Ec1655b and one from this function to the getter function at the address https://sepolia.etherscan.io/address/0x5fb1616F78dA7aFC9FF79e0371741a747D2a7F22, which returns the constant ```version = 4```.

**Reply by Robschuh** [reply]
Yeah, totally right, thanks!!
